### PR TITLE
create collection for styling z/m geometries

### PR DESCRIPTION
**Pull request by haubourg**
that collection offers a first line style
see  https://www.youtube.com/watch?v=o_lAeKJhRkg

more to come for 2.5 d improvements.

### CODE REVIEW

**Comment by elpaso** [comment]
Thanks Règis!
I did not test this PR yet but I have a few notes: you might want to provide more narrower preview images instead of a single one (that looks a bit too wide to me), the preview images will be presented in a nice ~ 4:3 slideshow to the users.

Also, you should add your collection metadata to: https://github.com/qgis/QGIS-Resources/blob/master/metadata.ini and add that to this PR.

**Comment by haubourg** [comment]
@elpaso. Hi alessandro, wow it seems I didn't save after cropping the preview. Should be fixed now

**Comment by DelazJ** [comment]
@elpaso (unless I miss it) maybe this repository needs a more verbose readme, mentioning where and what are the required files for a good contribution. Each of the previous PRs has a comment on a missing doc or a wrongly placed file.
Or maybe a template for the PR (like https://github.com/qgis/QGIS/pull/3919)

**Comment by elpaso** [comment]
@DelazJ I totally agree. 

We should add some links to the README.md and a PR template.
The documentation about the metadata is here: http://www.akbargumbira.com/qgis_resources_sharing/author/creating-metadata.html

BTW, I think the resource sharing plugin will eventually be part of core and the docs will need to be included in the official QGIS manual (AFAIK they are already in RST format).

**Comment by elpaso** [comment]
@DelazJ I've just added a link to the manual: https://github.com/qgis/QGIS-Resources/blob/master/README.md

**Comment by DelazJ** [comment]
@elpaso Thanks for the reactivity

File: metadata.ini [hunk 1]
```diff
@@ -20,3 +20,12 @@ description=The collection contains a complete resources to create a coherent ma
 preview=preview/osm_spatialite_googlemaps.png, preview/osm_spatialite_googlemaps_lines.qml.png
 qgis_minimum_version=2.0
 qgis_maximum_version=2.99
+
+[z_m_styling]
```
Comment (elpaso) [inline metadata.ini:24]: Can you please add the preview image path?

Comment (haubourg) [reply metadata.ini:24]: done!

**Comment by elpaso** [comment]
@haubourg thanks!

**Comment by anitagraser** [comment]
Thanks, great work!

I think this collection would also benefit from adding a sample dataset like I did in my recently published collection https://github.com/anitagraser/QGIS-resource-collections/tree/master/collections/conveyor_belt_flows. This way, users can download the style and immediately test it with the provided test dataset.

**Review by anitagraser — COMMENTED** [review]
Thanks for sharing! As mentioned already, would be great to also have a sample dataset.

**Comment by haubourg** [comment]
done in next PR!


2017-01-03 23:03 GMT+01:00 Anita Graser <notifications@github.com>:

**Pull request merged**